packages/angular/generators.json

```
@@ -207,6 +213,12 @@
       "aliases": ["mv"],
       "description": "Moves an Angular application or library to another folder within the workspace and updates the project configuration."
     },
+    "mfe-host": {
+      "factory": "./src/generators/mfe-host/mfe-host",
+      "schema": "./src/generators/mfe-host/schema.json",
+      "aliases": ["host"],
+      "description": "Generate a Host Angular Micro-Frontend App."
```

nit: use full words for `app`, it reads better in the docs
```suggestion
      "description": "Generate a Host Angular Micro-Frontend Application."
```